include/package-pack.mk

```diff
@@ -312,9 +312,11 @@ else
 		echo 'export root="$$$${IPKG_INSTROOT}"'; \
 		echo 'export pkgname="$(1)"'; \
 		echo "default_prerm"; \
-		[ ! -f $$(ADIR_$(1))/prerm-pkg ] || cat "$$(ADIR_$(1))/prerm-pkg"; \
+		[ ! -f $$(ADIR_$(1))/prerm-pkg ] || sed 's/^\s*//' "$$(ADIR_$(1))/prerm-pkg"; \
 	) > $$(ADIR_$(1))/pre-deinstall;
 
+	[ ! -f $$(ADIR_$(1))/postrm ] || sed -i 's/^\s*//' "$$(ADIR_$(1))/postrm";
```

Comment:
What is the purpose of posting postrm content here?

Reply:
This specifically is the bug fix.  When the `postrm` section is indented in the Makefile, it is then copied verbatim into the .apk, not suffixed onto some default code as in the other two cases (see the `post-deinstall:` sections of the examples, above).  This results in an indented shebang, which causes an error when executed during the `apk del` processing.